device-types/Fortinet/FG-100D.yaml

```diff
@@ -0,0 +1,66 @@
+---
+manufacturer: Fortinet
+model: FortiGate 100D
+slug: fg-100d
+part_number: FG-100D
+u_height: 1.0
+is_full_depth: false
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: DC Input for RPS
+    type: dc-terminal
+    maximum_draw: 63
+    allocated_draw: 53
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 63
+    allocated_draw: 53
+interfaces:
+  - name: dmz
+    type: 1000base-t
+  - name: ha1
+    type: 1000base-t
+  - name: ha2
+    type: 1000base-t
+  - name: mgmt
```

Comment:
If this port is management only, please include `mgmt_only: true`

Reply:
52cbe04aeb33e68afb1162cdca6173621fbf2d55